main.go

```diff
@@ -181,6 +186,18 @@ func mainErr(args []string) error {
 		os.Setenv("GARBLE_DIR", wd)
 		os.Setenv("GARBLE_LITERALS", fmt.Sprint(garbleLiterals))
 
+		if debugDir != "" {
+			if !filepath.IsAbs(debugDir) {
+				debugDir = filepath.Join(wd, debugDir)
+			}
+			err = testDebugDir(debugDir)
+			if err != nil {
```

Comment:
this can literally just be:

```
if info, err := os.Stat(debugDir); os.IsNotExist(err) {
    // MkdirAll ...
} else if err != nil {
    return fmt.Errorf... // other errors, e.g. can't access the path
} else if !info.IsDir() {
    return fmt.Errorf... // already exists and isn't a dir
}
```

Reply:
You mean remove the testdir function?

Reply:
Yes. you can do all of that with ~8 lines, as shown above. no need for a separate func.